ebml/EbmlUnicodeString.h

```diff
@@ -42,19 +42,17 @@ class EBML_DLL_API UTFstring {
   UTFstring & operator=(wchar_t);
 
   /// Return length of string
-  std::size_t length() const {return _Length;}
+  std::size_t length() const {return WString.size();}
 
-  explicit operator const wchar_t*() const;
-  const wchar_t* c_str() const {return _Data;}
+  explicit operator const wchar_t*() const {return WString.c_str();};
+  const wchar_t* c_str() const {return WString.c_str();}
```

Comment:
We might provide a direct access to the wstring in EbmlUnicodeString and UTFString. We have `explicit operator const std::string &() const;` for EbmlString. In C++ code that might be useful rather than using a `wchar_t *`.

This can be done in a separate MR.